docs/toc.md

```diff
@@ -289,13 +289,7 @@
 ## Tutorials
 ### [F# Interactive](fsharp/tutorials/fsharp-interactive/index.md)
 ### [Type Providers](fsharp/tutorials/type-providers/index.md)
-#### [Accessing a SQL Database by Using Type Providers](fsharp/tutorials/type-providers/accessing-a-sql-database.md)
-#### [Accessing a SQL Database by Using Type Providers and Entities](fsharp/tutorials/type-providers/accessing-a-sql-database-entities.md)
```

Comment:
what should happen to these topics that are out of the TOC now?

Reply:
I'm personally okay with deleting them, but there are probably some existing links out there which would break if they were deleted. If they're unlinked from the ToC, I would assume that this would give them less SEO weight. But if we still think they'll show up in searches, we should just delete them because they suggest using libraries which are out of date and unmaintained.

Reply:
SEO is not based on whether they show up on the TOC or not, so I think you should remove them. Links out there should be redirected to whatever replacement you put for them [here](https://github.com/dotnet/docs/blob/master/.openpublishing.redirection.json).

Reply:
Done!